modules/rental.rb

```diff
@@ -0,0 +1,19 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date)
+    @date = date
+    @book = book
+    @person = person
```

Comment:
- [x] Nice work creating @book and @person, however, The `rental` should add itself to the `@book` and @person` achieving the 'belongs-to' association

```suggestion
    @book = book
    @person = person
    book.rentals << self
    person.rentals << self
```